doc/changes/changes48.xml

```diff
@@ -0,0 +1,142 @@
+major &GAP; release made from this repository. The public issue tracker
+for the core &GAP; system is located at 
+<URL>https://github.com/gap-system/gap/issues</URL>,
+and you may use appropriate milestones from 
+<URL>https://github.com/gap-system/gap/milestones</URL> to see all changes
+that were introduced in corresponding &GAP; releases. An overview of the
+most significant ones is provided below.
+
+<Section Label="gap480"> 
+<Heading>&GAP; 4.8.0 (Autumn 2015)</Heading>
+
+<Subsection Label="Changes in the core GAP48 system"> 
+<Heading>Changes in the core &GAP; system introduced in &GAP; 4.8</Heading>
+
+Improved and extended functionality:
+<List>
+<Item>
+Unnecessary tests for the finiteness of the field were removed from 
+<Ref Func="GModuleByMats" BookName="ref"/> and related functions 
+(those that do not need MeatAxe, but just linear algebra).
+Thus the correponding functions will work in the case of characteristic 0
+or for other infinite fields, too.
+</Item> 
+<!-- https://github.com/gap-system/gap/pull/192 -->
+<Item>Extended support for accessing lists using multiple indices and
+indexing into lists using indices other than positive small integers.
+This allows library or packages to install methods supporting expressions 
+like
+<Log><![CDATA[
+m[1,2];
+m[1,2,3] := x;
+IsBound(m["a","b",Z(7)]);
+Unbind(m[1][2,3])
+]]></Log>
+</Item>
+<Item>
+The most common cases of <Ref Oper="AddDictionary" BookName="ref"/> 
+on three arguments now bypass method selection, avoiding the cost 
+of determining homogeneity for plain lists of mutable objects.
+</Item>
+<Item> 
+<!-- https://github.com/gap-system/gap/issues/7 -->
+Package authors may optionally specify the source code repository,
+issue tracker and support email address for their package using new
+components in the <File>PackageInfo.g</File> file, which will be used
+to create hyperlinks from the package overview page (see 
+<File>PackageInfo.g</File> from the Example package which you may use
+as a template).
+</Item>
+</List>
+
+Changed functionality:
+<List>  
+<Item>
+<!-- https://github.com/gap-system/gap/pull/90 -->
+Added support for partially variadic functions to allow function expressions
+like
+<Log><![CDATA[
+function(a,b,c,arg) ... end;
+]]></Log>
+which would require at least three arguments and assign the first three 
+to <A>a</A>, <A>b</A> and <A>c</A> and then a list containing any remaining 
+ones to <A>arg</A>.
+<P/>
+This may break an existing code when <A>arg</A> is used as a name of the
+argument which is not the last one.
+</Item>
+<Item>
+As a preparation for the future migration to the multithreaded &GAP;&nbsp;5,
+some language extensions from the <Package>HPC-GAP</Package> project were
+backported to the &GAP; library to help to unify the codebase of both 
+&GAP;&nbsp;4 and &GAP;&nbsp;5. At the moment, the only change which is not
+backwards compatible is <C>atomic</C> is no longer a valid identifier in 
```

Comment:
Replace second "is" by ", which is". But I'd actually rephrase this sentence:

"The only change which is not backwards compatible is that <C>atomic</C> now is a keyword, and thus is no longer a valid identifier. So if you have any variables or functions using that name, you will have to change it in &GAP; 4.8".